src/main/resources/templates/feedback.html

```diff
@@ -0,0 +1,198 @@
+{# Code eval page a peer reviewer will interact with #}
+{% extends "base/empty-page" %}
+
+{% block head %}
+<link rel="stylesheet" type="text/css" href="/css/highlightjs-tomorrow-night.css" />
+{% endblock %}
+
+{% block body %}
+<div class="eval">
+	<div class="left">
+		<div class="eval-header">
+			<div class="global-title-font">Simple Dependency Injection</div>
+			<div class="subtitle">Java Challenge</div>
+		</div>
+
+		<div class="eval-flag">
+			<div class="global-title-font">Grading Rubric</div>
+			<div class="eval-rubric">
```

Comment:
Very small thing, but based off the comments provided for the code submission, was a 13.5 warranted? Wouldn't it be a higher score?